### ISSUE TITLE
Special Exceptions to verbal noun syntax

### ISSUE THREAD

**Issue by elliottlash12** [elliottlash12/UD_Old_Irish-CritMinorGlossesMilan#13]
https://github.com/elliottlash12/UD_Old_Irish-CritMinorGlosses/blob/347e79ebca0256806ef07b4ea35c2817afb7cc66/sga_critminorglosses-ud-test.conllu#L986

Note that in this case the deprel associated with the preposition *do* needs to be *case* not *mark:prt* even though the following item is a verbal noun. 

Note too that the noun that is dependent on the verbal noun correctly gets the deprel *nmod* rather than *obj*. 

Basically, the verbal noun is not being analyzed as a verb here. How to tell this situation apart from the normal situation? Basically it comes down to the possible translation, I guess.
